demo/tutorials/task-specific-notebooks/Translation_Notebook.ipynb

```diff
@@ -0,0 +1,1253 @@
+        "])\n",
+        "\n",
+        "translation_model = translation_pipeline.fit(\n",
+        "    spark.createDataFrame([[\"\"]]).toDF(\"text\"))\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 15,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "yey-zVICtq-w",
+        "outputId": "9a933734-2a4c-4a9d-bbaf-9ef4cf562ea5"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Warning::Spark Session already created, some configs may not take.\n",
+            "recognize_entities_dl download started this may take some time.\n",
+            "Approx size to download 160.1 MB\n",
+            "[OK!]\n"
+          ]
+        }
+      ],
+      "source": [
+        "h = Harness(task=\"translation\",  model=translation_model,\n",
+        "            hub=\"johnsnowlabs\", data=\"Translation-test\")\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 16,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "JwK7oi7Etq-w",
+        "outputId": "41e7eafd-ab75-4fab-b401-ba45450dd1e0"
+      },
+      "outputs": [
+        {
+          "data": {
+            "text/plain": [
+              "{'tests': {'defaults': {'min_pass_rate': 0.65},\n",
+              "  'robustness': {'uppercase': {'min_pass_rate': 0.66},\n",
+              "   'lowercase': {'min_pass_rate': 0.6}}}}"
+            ]
+          },
+          "execution_count": 16,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "harness.configure(\n",
+        "    {\n",
+        "        'model_parameters': {'target_language': 'de'},\n",
+        "        'tests': {'defaults': {'min_pass_rate': 0.65},\n",
+        "                  'robustness': {'lowercase': {'min_pass_rate': 0.66},\n",
+        "                                 'uppercase': {'min_pass_rate': 0.66},\n",
+        "                                 }\n",
+        "                  }\n",
+        "    }\n",
+        ")\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "vTbPwStvtq-x",
+        "outputId": "4df4bccd-0046-4198-e71c-fcd32a9488c3"
+      },
+      "outputs": [],
+      "source": [
+        "harness.generate().run().report()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 270
+        },
+        "id": "AUZUeCpLtq-x",
+        "outputId": "c5d4327b-7ab7-4501-e4eb-1064e84ceb60"
+      },
+      "outputs": [],
+      "source": [
+        "harness.report()\n"
```

Comment:
We can show harness.generated_results() here, as report is already printed in above cell.